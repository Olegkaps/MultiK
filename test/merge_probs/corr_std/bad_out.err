ERROR: can't open file "nodir/nofile".
