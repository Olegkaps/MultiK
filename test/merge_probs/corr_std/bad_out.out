Reading first file from: input/spline_down.txt
Reading second file from: input/spline_up.txt
Output file will be: nodir/nofile
